### PR TITLE
Moduling project

### PR DESCRIPTION
**Pull request by Arthasasyan**
The only actual changes to code:
1. Changed goal for travis and `run_tests_in_container.sh`, because tests couldn't be run without packaging.
2. Added slf4j dependency to parent `pom.xml`, because slf4j logger was missing.

### CODE REVIEW

**Comment by Arthasasyan** [comment]
@raipc should default goal in `pom.xml` be changed?

File: pom.xml [hunk 1]
```diff
@@ -147,39 +157,30 @@
...
-            </exclusions>
-        </dependency>
 
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
             <version>1.16.18</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```
Comment (raipc) [inline pom.xml]: Plugin versions should be specified in plugins section

File: pom.xml [hunk 2]
```diff
@@ -53,9 +54,20 @@
             <plugins />
         </pluginManagement>
         <plugins>
+            <plugin>
```
Comment (raipc) [inline pom.xml]: How is this plugin used?

File: atsd-jdbc-acceptance-tests/pom.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>atsd-jdbc-test</artifactId>
+        <groupId>com.axibase</groupId>
+        <version>0.5.0-SNAPSHOT</version>
```
Comment (raipc) [inline atsd-jdbc-acceptance-tests/pom.xml]: Use revision variable

File: atsd-jdbc-acceptance-tests/pom.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>atsd-jdbc-acceptance-tests</artifactId>
+    <packaging>jar</packaging>
+
+    <prerequisites>
```
Comment (raipc) [inline atsd-jdbc-acceptance-tests/pom.xml]: Remove prerequisites from child poms

File: atsd-jdbc-examples/pom.xml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    <artifactId>atsd-jdbc-examples</artifactId>
+    <packaging>jar</packaging>
+
+    <prerequisites>
+        <maven>3.5.0</maven>
+    </prerequisites>
+
+
+    <dependencies>
+        <dependency>
```
Comment (raipc) [inline atsd-jdbc-examples/pom.xml]: Examples should not depend on other tests

File: atsd-jdbc-examples/src/test/resources/logback-test.xml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        <encoder>
+            <Pattern>%d[%4.4t]%-5p %-21.21logger{20}: %m%n</Pattern>
+        </encoder>
+    </appender>
+ 
+    <logger name="com.axibase.tsd.driver.jdbc">
+        <level value="DEBUG" />
+    </logger>
+    
+    <logger name="com.zaxxer.hikari">
```
Comment (raipc) [inline atsd-jdbc-examples/src/test/resources/logback-test.xml]: Remove hikari and spring boot configuration from non-spring-boot modules

File: pom.xml [hunk 7]
```diff
@@ -147,39 +157,30 @@
         <dependency>
```
Comment (raipc) [inline pom.xml]: Remove Spring Boot dependencies from parent pom

File: atsd-jdbc-examples/pom.xml (hunk 5 above)
Comment (Arthasasyan) [reply atsd-jdbc-examples/pom.xml]: All dependencies on `acceptance-tests` were deleted. TestUtil dependencies cannot be deleted.

File: pom.xml (hunk 1 above)
Comment (Arthasasyan) [reply pom.xml]: Project is not built without specifying version 
```java
[ERROR] 'dependencies.dependency.version' for org.apache.maven.plugins:maven-compiler-plugin:jar is missing. @ com.axibase:atsd-jdbc-test:${revision}, /home/akost/atsd-jdbc-test/pom.xml, line 170, column 21
```

File: pom.xml (hunk 2 above)
Comment (Arthasasyan) [reply pom.xml]: It was copied from `aggregation-log-filter`, but it seems that it does not affect the result. I deleted it.

File: atsd-jdbc-acceptance-tests/pom.xml (hunk 3 above)
Comment (Arthasasyan) [reply atsd-jdbc-acceptance-tests/pom.xml]: IDEA says that using properties from parent module is prohibited, but project is built with it. Should `revision` variable be remained? 
image

**Pull request merged**